docs/faq/stream-to.md

```diff
@@ -48,3 +47,47 @@ You can indirectly stream to a device which only accepts a URL such as a Web Rad
 - Use the HA media player plugin in music assistant and select mpd as the output
 
 Thanks to [Manuel Rüger](https://github.com/mrueg) who showed us [here](https://github.com/orgs/music-assistant/discussions/2410#discussioncomment-10885780)
+
+## A device I am yet to purchase!
+
+A summary of the capabilities of the player providers available in Music Assistant [is available here](../player-support/index.md).
+
+In general terms the protocols/devices that should give you minimum to no setup difficulty are:
+
+-	Any player/speaker that supports [Airplay](https://en.wikipedia.org/wiki/AirPlay)
+-	Any speaker that supports the [Google Cast protocol](https://en.wikipedia.org/wiki/Google_Cast) (also known as cast builtin)
+-	[Squeezebox hardware](https://en.wikipedia.org/wiki/Squeezebox_(network_music_player))
+-	Software squeezebox/slimproto based players (e.g. [squeezelite](https://sourceforge.net/projects/lmsclients/files/squeezelite/) )
+-	[Sonos](https://www.sonos.com/en-us/home)
+
+We don’t believe most people can hear the difference in sample rates above CD quality so AirPlay is highly recommended. It has a good sync protocol and is widely implemented in consumer devices. For most people they should consider what they already have and fit in with that (unless it’s DLNA then consider changing due to the quirks of some devices) and their budget.
+
+Bluesound is shown in the [player provider table](../player-support/index.md) as not supporting sync. That is with the current MA implementation which should change in the future. For clarity, Bluesound devices do support sync.
+
+Note that many Sonos devices can be synced with AirPlay devices which is another plus for AirPlay.
+
+The following table is a non-exhaustive list of possible solutions:
+
+| Device or Software	          | Price# | Supported Protocols              |+Amp^| Pros and Cons |
+|-------------------------------|--------|----------------------------------|------|---------------|
+|[PiCorePlayer*](https://www.picoreplayer.org/)|$	     |Squeezelite, AirPlay, Bluetooth	  |  Y	 |Pros: Cheap, Runs on RPi 1<br>Cons: Requires some technical knowledge to install the free software|
+|[WiiM Pro](https://www.wiimhome.com/wiimpro/overview)|$$      |AirPlay 2, Google Cast Audio, DLNA|  Y   |Pros: Minimal setup, versatile<br>Cons: Cheaper options available, Cast requires app to have sync’ed group|
```

Comment:
wiim pro and ultra also supports squeezelite

Reply:
That isn't listed here https://www.wiimhome.com/wiimpro/overview ??

Reply:
Actually found it here https://faq.wiimhome.com/en/support/solutions/articles/72000610226-how-to-stream-music-from-lms-to-your-wiim-device-with-squeezelite